src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
```

Comment:
CommentList component import is missing file extension. Change the import statement to `import { CommentList } from '../CommentList/CommentList.tsx';`